internal/validators/oci_validator_test.go

```diff
@@ -294,14 +162,28 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEKPuCo9AmJCpqGWhefjbhkFcr1GA3
 iNa765seE3jYC3MGUe5h52393Dhy7B5bXGsg6EfPpNYamlAEWjxCpHF3Lg==
 -----END PUBLIC KEY-----`),
 			},
+			sv: v1alpha1.SignatureVerification{
+				SecretName: "secret",
+			},
 			expectedDetail: "no matching signatures were found",
 			expectErr:      true,
 		},
 	}
 
 	for _, tc := range testCases {
+		fmt.Println(tc.name)
```

Comment:
instead of doing an `fmt.Println(tc.name)` can you switch it to something like this:
```
for _, tc := range testCases {
    t.Run(tc.name, func(t *testing.T) {
        // rest of the test
    }
}
```
That should handle printing test names for us